src/llmq/instantsend.cpp

```diff
@@ -345,7 +345,9 @@ uint256 CInstantSendDb::GetInstantSendLockHashByTxid(const uint256& txid) const
     LOCK(cs_db);
     uint256 islockHash;
     if (!txidCache.get(txid, islockHash)) {
-        db->Read(std::make_tuple(DB_HASH_BY_TXID, txid), islockHash);
+        if (!db->Read(std::make_tuple(DB_HASH_BY_TXID, txid), islockHash)) {
+            return {};
```

Comment:
I wonder if it makes more sense to return an std::optional<uint256> as opposed to returning a null / empty value here... I think that would potentially be a less error prone api

Reply:
This feels a bit out of scope imo.